Generation/src/components/MomentumRangeParticleGun.cpp

```diff
@@ -9,12 +9,7 @@
 #include "GaudiKernel/PhysicalConstants.h"
 #include "GaudiKernel/IRndmGenSvc.h" 
 
-//===========================================================================
-// Implementation file for class: MomentumRangeParticleGun
-//
-// 2008-05-18: Patrick Robbe adaptation to tool structure
-// 2014-06-22: Benedikt Hegner adaption for non LHCb use cases
-//===========================================================================
```

Comment:
If we keep it, we should probably make it at least doxygen-readable. Do you maybe know of a suitable command to track a change + corresponding author?